code/modules/mob/living/carbon/human/blood.dm

```diff
@@ -88,26 +90,46 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 				if(!pale)
 					pale = 1
 					update_body()
-					var/word = pick("dizzy","woozy","faint")
-					src << "<span class='warning'>You feel [word].</span>"
-				adjustOxyLoss((BLOOD_VOLUME_NORMAL - blood_volume) / 100)
+				if(prob(15))
+					add_medical_effect(/datum/medical_effect/shock)
+				if(prob(25))
+					var/effect = pick(1,0)
+					if(effect)
+						Weaken(3)
+					else
+						Paralyse(3)
 			if(BLOOD_VOLUME_BAD to BLOOD_VOLUME_OKAY)
 				if(!pale)
 					pale = 1
 					update_body()
-				adjustOxyLoss((BLOOD_VOLUME_NORMAL - blood_volume) / 50)
-				if(prob(5))
-					blur_eyes(6)
-					var/word = pick("dizzy","woozy","faint")
-					src << "<span class='warning'>You feel very [word].</span>"
+				if(prob(25))
+					add_medical_effect(/datum/medical_effect/shock)
+				if(prob(40))
+					var/effect = pick(1,0)
+					if(effect)
+						Weaken(4)
+					else
+						Paralyse(4)
 			if(BLOOD_VOLUME_SURVIVE to BLOOD_VOLUME_BAD)
-				oxyloss += 5
-				if(prob(15))
-					Paralyse(rand(1,3))
-					var/word = pick("dizzy","woozy","faint")
-					src << "<span class='warning'>You feel extremely [word].</span>"
+				if(prob(50))
+					add_medical_effect(/datum/medical_effect/shock)
+				if(prob(50))
+					var/effect = pick(1,0)
+					if(effect)
+						Weaken(5)
+					else
+						Paralyse(5)
 			if(0 to BLOOD_VOLUME_SURVIVE)
-				death()
+				if(prob(75))
+					add_medical_effect(/datum/medical_effect/shock)
```

Comment:
Actually no blood should straight up kill you unless you're being cryogenically frozen or some shit